examples/refrigerator-app/silabs/include/RefrigeratorUI.h

```diff
@@ -0,0 +1,48 @@
+/*
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include "RefrigeratorIcons.h"
+#include "glib.h"
+#include "lcd.h"
+
+class RefrigeratorUI
+{
+
+public:
+    enum SUPPORTED_MODES
```

Comment:
use enum class, but more importantly, you re repeating this enum from RefrigeratiorManager.h. Use the same don't duplicate

Reply:
removed the enum